src/libs/registry/docset.cpp

```
@@ -277,6 +295,20 @@ QString Docset::documentPath() const
     return QDir(m_path).filePath(QStringLiteral("Contents/Resources/Documents"));
 }
 
+bool Docset::isArchived() const
+{
+    return m_tarixArchive != nullptr;
+}
+
+std::optional<QByteArray> Docset::readDocument(const QString &path) const
```

Cppcheck (2.20.0)

[style] The function 'readDocument' is never used.

(unusedFunction)